test/NuGet.Core.Tests/NuGet.CommandLine.Xplat.Tests/Utility/XPlatTestLogger.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using NuGet.CommandLine.XPlat;
+using NuGet.Common;
+using NuGet.Test.Utility;
+
+namespace NuGet.CommandLine.Xplat.Tests.Utility
+{
+    internal class XPlatTestLogger : CommandOutputLogger
+    {
+        private readonly bool _observeLogLevel;
+
+        public TestLogger Logger { get; set; } = new TestLogger();
+
+        public XPlatTestLogger(bool observeLogLevel = false)
+            : base(LogLevel.Debug)
+        {
+            _observeLogLevel = observeLogLevel;
+        }
+
+        protected override void LogInternal(LogLevel logLevel, string message)
+        {
+            if (_observeLogLevel && logLevel < VerbosityLevel)
+            {
+                return;
+            }
+
+            switch (logLevel)
+            {
+                case LogLevel.Debug:
+                    Logger.LogDebug(message);
+                    break;
+                case LogLevel.Error:
+                    Logger.LogError(message);
+                    break;
+                case LogLevel.Information:
+                    Logger.LogInformation(message);
+                    break;
+                case LogLevel.Minimal:
+                    Logger.LogMinimal(message);
+                    break;
+                case LogLevel.Verbose:
+                    Logger.LogVerbose(message);
+                    break;
+                case LogLevel.Warning:
+                    Logger.LogWarning(message);
+                    break;
+                default:
+                    Logger.LogDebug(message);
+                    break;
+            }
+        }
+
+        public override void LogMinimal(string data, ConsoleColor color)
+        {
+            LogInternal(LogLevel.Minimal, data);
+        }
+
+        public ConcurrentQueue<string> Messages
+        {
+            get
+            {
+                return Logger.Messages;
+            }
+        }
+
+        public ConcurrentQueue<string> ErrorMessages
+        {
+            get
+            {
+                return Logger.ErrorMessages;
+            }
+        }
```

Comment:
```suggestion
        public ConcurrentQueue<string> ErrorMessages => Logger.ErrorMessages;
```

or

```suggestion
        public ConcurrentQueue<string> ErrorMessages
        {
            get => Logger.ErrorMessages;
        }
```

Reply:
This logger was no longer being used after some refactoring that I did, so I've removed the file entirely now